{{cookiecutter.project_slug}}/_/frameworks/SpringBoot/pom.xml

```diff
@@ -36,6 +36,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-actuator</artifactId>
+        </dependency>
```

Comment:
As explained by @ZuGao yesterday, this is a useful addition to the Spring Boot setup to easily provide health check endpoints for the Kubernetes deployment configuration.

Noteworthy to remember for all of us! :point_up: